go/test/endtoend/backup/vtctlbackup/backup_utils.go

```diff
@@ -998,7 +998,7 @@ func verifySemiSyncStatus(t *testing.T, vttablet *cluster.Vttablet, expectedStat
 }
 
 func terminateBackup(t *testing.T, alias string) {
-	stopBackupMsg := "Done taking Backup"
+	stopBackupMsg := "finished restore:"
```

Comment:
I don't understand this. The test is calling `Backup` on line 1010. Why would we expect a `finished restore` message?

Reply:
Good catch, updated the log.